.eslintrc.json

```diff
@@ -12,6 +12,7 @@
   },
   "rules": {
     "one-var": 0,
+    "no-unused-expressions": 0,
```

Comment:
@chingsley why are we ignoring this?

Reply:
I was testing a parameter that equals null, and Eslint flagged it as "unused-expression". But I have found another way to test that parameter, so I will remove this setting from the file.